### PR TITLE
Setup and mobile skeleton feature

### PR DESCRIPTION
**Pull request by Ursaaaaaa**
Began my first Microverse project for my Portfolio
1. Selected a template from Figma
2. Added README file
3. Added HTML and CSS
4. Updated HTML components
5. Updated CSS components 
6. Updated background image
7. Updated and imported Google fonts

### CODE REVIEW

File: portfolio.html [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    <link rel="stylesheet" href="mystyle.css" />
+  </head>
+  <body>
+    <header>
+      <div class="logo">Nii Amanor Djoleto</div>
+      <ul>
+        <li><a href="#Portfolio">Portfolio</a></li>
+        <li><a href="#About"> About</a></li>
+        <li><a href="#Contact"> Contact</a></li>
+      </ul>
```
Comment (BenMukebo) [inline portfolio.html]: - [ ] Please wrap this section in a nav tag, and add the hamburger to it,  this is supposed to be your navbar items, see these images below to compare your design to the original one

- [ Your design] 
- Please use correct adjustment  of `padding, margin, font-weight, font-size and color` as per the original design

bbbb

- Now it is time to check this image below and make changes to your actual design to much it, please fix that and you will be goo to go ⚒️, [here](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/1_setup.md) the project requirement you should consider for this project.

- [ FIGMA]

FIMAAAAA

File: portfolio.html [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Portfolio</title>
+    <link rel="stylesheet" href="mystyle.css" />
+  </head>
+  <body>
+    <header>
+      <div class="logo">Nii Amanor Djoleto</div>
+      <ul>
```
Comment (BenMukebo) [inline portfolio.html]: - [x] Please wrap this your logo in an `a` tag, remember, the user will use this logo link to get back to the home page
- [ Optional] the name you have chosen is too long for a logo, I will suggest you make it a bit short 😄

**Review by BenMukebo — CHANGES_REQUESTED** [review]
### Hi *** @ramrod433 , ***

You did a great job so far 👏🏼 👏🏼
There are some issues that you still need to work on to go to the next project but you are almost there!

### To highlight 💪🏻

- No linter errors. ✔️ 
- Github flow was correctly used.  ✔️   
- Good commit messages ✔️ 
- Descriptive README file.  ✔️ 
 👍  Good job!


## STATUS: CHANGES REQUESTED  ♻️ 
_Please Check the comments under the review and make all required changes_



## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding!👏👏👏 💻  

- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me on( @BenMukebo ) in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: portfolio.html [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+      <div class="info">
+        <h1>Hey there, I am Nii Amanor</h1>
+        <h2>I am a Software Developer</h2>
+        <p>
+          I can help you build a product , feature or website Look through some
+          of my work and experience! If you like what you see and have a project
+          you need coded, don’t hestiate to contact me.
+        </p>
+      </div>
+    </section>
```
Comment (thecodechaser) [inline portfolio.html]: - [ ] Good job working on the changes requested by the previous reviewer, While implementing the changes you introduce issues to your design
 - Your background images overflow to the hero section content, Please use an only required background image(the image at the right bottom in the Figma template) and make sure it matches the design on the [Template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A1471).
 - You removed the social media icons from the design, Please place the social media icons and make sure it matches the design on the [Template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A1471). 

### Your design

image

### Figma design
image

**Review by thecodechaser — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ 

## Hi @ramrod433 👋, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

## To Highlight 🎉
- No linter errors. :heavy_check_mark:
- GitHub flow is followed. :heavy_check_mark:
- Professional README file. :heavy_check_mark:
- Good commit messages. **✔️**

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: portfolio.html (hunk 3 above)
Comment (Ursaaaaaa) [reply portfolio.html]: @thecodechaser please can you give advice on how to make the background image work

Comment (thecodechaser) [reply portfolio.html]: > @thecodechaser please can you give advice on how to make the background image work

@ramrod433 Hello, There are many ways to go but I would suggest You can use `position: absolute` for the image and place it where you want with top, bottom, right, left properties.

Comment (Ursaaaaaa) [reply portfolio.html]: @thecodechaser thank you very much

File: mystyle.css [hunk 4]
```diff
@@ -0,0 +1,137 @@
...
+  position: ;
+  left: 65.74%;
+  right: -30.09%;
+  top: 29.3%;
+  bottom: -11.63%;
+  transform: matrix(-0.96, 0.27, 0.27, 0.96, 0, 0);
+  display: flex;
+  flex-direction: row-reverse;
+} */
+
```
Comment (Danieldotcomcoder) [inline mystyle.css:87]: [OPTIONAL] Kindly remove any commented out code, it will make your code more professional . 👍

File: mystyle.css [hunk 5]
```diff
@@ -0,0 +1,137 @@
+@import url("https://fonts.googleapis.com/css2?family=Crete+Round&display=swap");
```
Comment (Danieldotcomcoder) [inline mystyle.css:1]: [OPTIONAL]: There is a horizontal scroll in your design, It will look better if you fix it  😃👍

**Review by Danieldotcomcoder — APPROVED** [review]
Hello @ramrod433  , 

Your project is complete!  There is nothing else to say other than... it's time to merge it 👍 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Ursaaaaaa** [comment]
Thank you soo much

**Pull request merged**